src/tallies/tally.cpp

```diff
@@ -250,6 +254,42 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), 1001) !=
+                std::end(scores_);
+  if (exists == 1) {
+    std::cout << "PULSE HEIGHT TALLY SIMULATION, VERSION 1.0, 15.09.2021"
```

Comment:
Probably shouldn't have separate versions on individual tallies. The commit history will tell us all we need to know.

Reply:
yes, I removed it.